opensrp-reveal/src/main/java/org/smartregister/reveal/interactor/BaseFormFragmentInteractor.java

```diff
@@ -81,5 +91,34 @@ public void findMemberDetails(String structureId, JSONObject formJSON) {
         });
     }
 
+    @Override
+    public void findSprayDetails(String interventionType, String structureId, JSONObject formJSON) {
```

Comment:
This method looks is a duplicate of `org.smartregister.reveal.interactor.ListTaskInteractor#getSprayDetails`, maybe you can move the method to a utils, or common class. Also it can just return  CommonPersonObject